src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,50 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(fromFileName, toFileName);
+
+    }
 
+    private HashMap<String, Integer> readFromFile(String readFrormFile) {
+        HashMap<String, Integer> statistic = new HashMap<String, Integer>();
```

Comment:
HashMap is too heavy data structure, you can just return int[] in this method and provide it to createReport()